src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  sourceString.replace(/;\s*;/g, ' ').trim();
```

Comment:
This line attempts to replace multiple semicolons with a single space and trim the string, but the result of `replace` is not being used because it does not modify the original string. You need to assign the result to a variable or directly to `sourceString`.